File: Data Structures & Algorithms/concatenation-of-array/submission-2.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] nums1 = new int[2 * nums.length];

        for (int i = 0; i < nums1.length; i++) {
            if (i < nums.length) {
                nums1[i] = nums[i];
            } else {
                nums1[i] = nums[i - nums.length];
            }
        }

        return nums1;
        
    }
}

File: Data Structures & Algorithms/find-duplicate-integer/submission-2.py
class Solution:
    def findDuplicate(self, nums: List[int]) -> int:
        mp = {}
        for n in nums:
            if n not in mp:
                mp[n] = 1
            else:
                mp[n] += 1
            
            if mp[n] > 1:
                return n
        
        return -1